src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IParameterReferenceExpression.vb

```diff
@@ -659,7 +660,11 @@ End Class]]>.Value
 
             Dim expectedOperationTree = <![CDATA[
 IInvalidStatement (OperationKind.InvalidStatement, IsInvalid) (Syntax: 'Case x')
```

Comment:
Should this node be marked as Implicit?

Reply:
you mean all invalid statement should be marked as implicit?

Reply:
> you mean all invalid statement should be marked as implicit?

That is not what I mean. It looks like we already have a child of this node associated with the same syntax. Either there this is not the case and we at least should output syntax kind for the nodes so that the reader can distinguish nodes that cannot be distinguished simply by looking at their ToString() result, or one of the nodes is synthesized and should be marked Implicit.